base_layer/wallet_ffi/src/lib.rs

```diff
@@ -4619,6 +4619,65 @@ pub unsafe extern "C" fn comms_list_connected_public_keys(
     }
 }
 
+/// Gets the length of the public keys vector
+///
+/// ## Arguments
+/// `*const TariPublicKeys` - Pointer to emoji set
+///
+/// ## Returns
+/// `c_uint` - Length of the TariPublicKeys vector, 0 if is null
+///
+/// # Safety
+/// None
+#[no_mangle]
+pub unsafe extern "C" fn public_keys_get_length(public_keys: *const TariPublicKeys, error_out: *mut c_int) -> c_uint {
+    let mut error = 0;
+    ptr::swap(error_out, &mut error as *mut c_int);
+    if public_keys.is_null() {
+        error = LibWalletError::from(InterfaceError::NullError("public_keys".to_string())).code;
+        ptr::swap(error_out, &mut error as *mut c_int);
+        return 0;
+    }
+    (*public_keys).0.len() as c_uint
+}
+
+/// Gets a ByteVector at position in a EmojiSet
```

Comment:
public keys?

Reply:
These are indeed public keys. It's the set of connected base nodes.